internal/store/store.go

```diff
@@ -268,10 +270,27 @@ func (s *Store) setAtlasClient(client *http.Client) error {
 	if s.baseURL != "" {
 		opts = append(opts, atlas.SetBaseURL(s.baseURL))
 	}
+	if log.IsDebugLevel() {
+		opts = append(opts, atlas.SetWithRaw())
+	}
 	c, err := atlas.New(client, opts...)
 	if err != nil {
 		return err
 	}
+	c.OnResponseProcessed(func(resp *atlas.Response) {
+		respHeaders := ""
+		for key, value := range resp.Header {
+			respHeaders += fmt.Sprintf("%v: %v\n", key, strings.Join(value, " "))
+		}
+
+		_, _ = log.Debugf(`request:
+%v %v
+response:
+%v %v
+%v
+%v
+`, resp.Request.Method, resp.Request.URL.Path, resp.Proto, resp.Status, respHeaders, string(resp.Raw))
```

Comment:
@andreaangiolillo it is not impossible 😅

This is a slice of bytes, so it would take some reparsing, I'll leave it for now we can always open a follow up ticket.